en/docs/develop/apis/identity-governance-rest-api.md

```diff
@@ -152,6 +124,65 @@ and properties of connectors as mentioned below. <br>
     | HTTP Connection Request Timeout | adaptive_authentication.analytics.HTTPConnectionRequestTimeout | HTTP Connection Request Timeout in milliseconds |
     | Hostname verification | adaptive_authentication.analytics.hostnameVerfier | Hostname verification. (STRICT, ALLOW_ALL) |
 
+    **User Claim Update**
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Enable user email verification on update | UserClaimUpdate.Email.EnableVerification | Trigger a verification notification when user's email address is updated. |
+    | Email verification on update link expiry time | UserClaimUpdate.Email.VerificationCode.ExpiryTime | Validity time of the email confirmation link in minutes. |
+    | Enable user mobile number verification on update | UserClaimUpdate.MobileNumber.EnableVerification | Trigger a verification SMS OTP when user's mobile number is updated. |
+    | Mobile number verification on update SMS OTP expiry time | UserClaimUpdate.MobileNumber.VerificationCode.ExpiryTime | Validity time of the mobile number confirmation OTP in minutes. |
+
+??? "Account Management"
+
+    **Idle Account Suspend**
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Suspend idle user accounts   | suspension.notification.enable   | Lock user account after a given idle period. |
+    | Allowed idle time span in days   | suspension.notification.account.disable.delay   | Time period in days before locking the user account. |
+    | Alert sending time periods in days   | suspension.notification.delays   | Send warning alerts to users before locking the account, after each period. Comma separated multiple values accepted. |
+
+    **Account Disable**
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Enable account disabling   | account.disable.handler.enable   | Allow an administrative user to disable user accounts |
+    | Manage notification sending internally   | account.disable.handler.notification.manageInternally   | Disable, if the client application handles notification sending |
+    
+    **Account Recovery**
+    
+    | Property display name | Property name | Purpose |
+    | ------- | -------|---|
+    | Notification based password recovery   | Recovery.Notification.Password.Enable   |  |
```

Comment:
Don't these need the purpose field?

Reply:
these weren't returned in the API responses. since we discussed offline that one to one mapping is not required here, I will add it.